autometa/common/external/diamond.py

```diff
@@ -397,6 +397,7 @@ def main(args):
 
 if __name__ == '__main__':
     import argparse
+    import os
```

Comment:
It is used to run the `default=os.curdir` in `parser.add_argument('--tmpdir', help='</path/to/tmp/directory>', default=os.curdir)`

Reply:
I wanted to make the `argparse` block standalone so that we can easily copy it to another file and run it.
If I remove `import os` and copy the `arparse` block and try to run it it gives me an error.

Reply:
You could `import os` in the written temporary file without having to move this in the source code